content/_help/verify-your-identity/accepted-state-issued-identification._en.md

```diff
@@ -1,33 +1,28 @@
 ---
 layout: help
-title: Accepted state-issued identification
+title: Accepted identification documents
 category: verify-your-identity
 permalink: /help/verify-your-identity/accepted-state-issued-identification/
 order: 2
 redirect_from:
   - /en/help/verify-your-identity/accepted-state-issued-identification/
+do_list: 
+  -  "**Driver’s license** from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, American Samoa, Mariana Islands and Puerto Rico)."
+  - "**Non-driver’s license state-issued ID card.** This is an identity document issued by the state, the District of Columbia (DC), or US territory that asserts identity but does not give driving privileges."
+dont_list:
+  - You cannot pass identity verification if your ID is expired.
+  - You cannot use extension documents in place of an unexpired&#160;ID.
+  - You cannot use a paper or temporary ID.
+alert: "**If you do not have a valid drivers' license or state ID card, you cannot use Login.gov for identity verification.** Please contact the partner agency’s help center to find out what you can do instead."
 ---
 
-At this time, only the following valid state-issued identification is accepted: 
+{% include components/icon-list.html items=page.do_list size='md' icon_color='green' icon_shape='check_circle'  style_fix='padding-left: 0 !important;' %}
 
-* Driver’s license from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, American Samoa, Mariana Islands and Puerto Rico)
-* A non-driver’s license state-issued ID card
+### We do not accept military IDs, U.S. Passports, or other identification documents.
 
-  * This is an identity document issued by the state, the District of Columbia (DC), or US territory that asserts identity but does not give driving privileges.
+{% include components/alert.html text=page.alert role='alert' variant='error' %}
 
-**What if my driver’s license or state-issued ID card is expired?**
-You cannot pass identity verification if your ID is expired. Extension documents cannot be used in place of a current, unexpired ID.
-
-**Are paper or temporary IDs accepted?**
-No, you cannot use a paper or temporary ID. 
-
-**What if my ID is too worn or damaged?**
-You cannot pass identity verification if your ID is too worn or damaged. 
-
-**What if my current address is different from the address on my ID?**
-You can change your address on the ‘verify’ step after you enter your Social Security number. The address will need to match an address on record for you. 
-
-If you do not have state-issued identification or cannot complete this step, please contact the partner agency’s help center to find out what you can do.
+{% include components/icon-list.html items=page.dont_list size='md' icon_color='red' icon_shape='cancel' style_fix='padding-left: 0 !important;'%}
```

Comment:
I had previously chatted with @charleyf and @kellular about this proposed padding customization to the component, and it was my understanding that it wasn't expected to be different from the default styles. So I wouldn't expect us to be customizing the left padding as we're doing here.

Reply:
@aduth , ok removed.